src/Backend/DataSeriesFactory.js

```diff
@@ -0,0 +1,27 @@
+export default class TraceFactory{
+    static getXAxisByPoints(start, end, points){
+        const interval = (end-start)/points
+        return this.getXAxis(start,end, points,interval)
+    }
+
+    static getXAxisByInterval(start,end,interval){
+        const points = Math.ceil((end-start)/interval)
+        return this.getXAxis(start,end,points,interval)
+    }
+
+    static getXAxis(start,end,points,interval){
```

Comment:
What is the purpose of this method ?

Reply:
`points` is unclear - is it number of points or list of points ?
If this is number of points, then call it differently.

Also - how this will behave if we have:
 - start  = 0
 - stop = 100
 - points = 20
 - interval = 1
 - 
What if will produce ?

Reply:
It's basically a private method of this class, but react doesn't support private class member methods as they are an experimental feature. Calling the function with given values will produce a series of 20 points - 19 separated by the interval with the last point being the end point. But since it's technically private it should never be called with those params as the other one is calculated aboved based on the other

Reply:
And the purpose is to actually generate X axis values as oposed to getXAxisByPoints and getXAxisByInterval which calculate the other parameter and call this function. Maybe we could use '_' to signify that it's private then? And also change the name to "generateXAxisValues"